openpype/hosts/substancepainter/api/lib.py

```diff
@@ -643,3 +643,55 @@ def _setup_prompt():
         return
 
     return project_mesh
+
+
+def has_rgb_channel_in_texture_set(texture_set_name, map_identifier):
+    """Function to check whether the texture has RGB channel.
+
+    Args:
+        texture_set_name (str): Name of Texture Set
+        map_identifier (str): Map identifier
+
+    Returns:
+        colorspace_dict: A dictionary which stores the boolean
+            value of textures having RGB channels
```

Comment:
It does not return a `dict`
```suggestion
    Returns:
        bool: Whether the channel type identifier has RGB channel or not
            in the texture stack.
```